cocos/core/gfx/device-manager.ts

```diff
@@ -27,7 +28,7 @@ import { JSB } from 'internal:constants';
 import { legacyCC } from '../global-exports';
 import { error, getError } from '../platform/debug';
 import { sys } from '../platform/sys';
-import { BindingMappingInfo, DeviceInfo, SwapchainInfo } from './base/define';
+import { BindingMappingInfo, DeviceInfo, SwapchainInfo } from '.';
```

Comment:
Why change this line?

Reply:
still some circular dependencies in ts, thanks for figuring it out